packages/spartan/extra/dcos-spartan-watchdog.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
We already have a dns lib that we use in https://github.com/dcos/dcos/blob/master/packages/spartan/extra/gen_resolvconf.py

Actually, this and that seem like they're just entry points to the same "check if spartan is up" code.